src/tseda/vpages/ignn.py

```diff
@@ -122,25 +127,52 @@ def plot(self, haplotype=0):
         )
         return p
 
+    def check_inputs(self, inds):
+        try:
+            print(self.individual_id)
+            if (
+                not isinstance(self.individual_id, (int, float))
+                or self.individual_id < 0
+            ):
+                self.position_index_warning.visible = True
+                return None
+            else:
+                self.position_index_warning.visible = False
+                nodes = inds.loc[self.individual_id].nodes
+                return nodes
+        except KeyError:
+            self.position_index_warning.visible = True
+            return None
+
     @pn.depends("individual_id", "window_size")
     def __panel__(self, **params):
         inds = self.datastore.individuals_table.data.rx.value
         if self.individual_id is None:
-            return pn.pane.Markdown("")
-        nodes = inds.loc[self.individual_id].nodes
-        return pn.Column(
-            pn.pane.Markdown(f"## Individual id {self.individual_id}"),
-            self.warning_pane,
-            pn.pane.Markdown(f"### Haplotype 0 (sample id {nodes[0]})"),
-            self.plot(0),
-            pn.pane.Markdown(f"### Haplotype 1 (sample id {nodes[1]})"),
-            self.plot(1),
-        )
+            return pn.pane.Markdown(
+                "**Enter a valid sample id to see the GNN haplotype plot.**"
```

Comment:
Maybe we should still display the heads "GNN Haplotype plot" so that the user understands what plot this message is connected to, i.e. not the GNN VBar plot

Reply:
Done!